test/client/fixtures/sandbox/node/window-test.js

```diff
@@ -298,6 +298,18 @@ if (window.history.replaceState && window.history.pushState) {
     });
 }
 
+test('overridden "target" attribute getter must return same value as origin getter for elements without attribute', function () {
```

Comment:
'an overridden "target" attribute getter should return the same value as origin getter (attribute value is not defined)'